index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles/indexStyles.css">
+    <title>Document</title>
+</head>
+<body>
+    <div class="grand_container__styles">    
+        <div class="header_container__styles">
+            <h1 class="header_h1__styles">WHO WE ARE</h1>
```

Comment:
Better to omit the usage of HTML elements names inside classes
something like `header_h1__styles ===> header_heading-lvl-one__styles`